source/_patterns/01-elements/textarea/textarea.scss

```diff
@@ -29,6 +30,10 @@
 	// align-items: center;
 	// justify-content: flex-start;
 
+	.elm-label:has(+ &) {
```

Comment:
This results in `.elm-label:has(+ .elm-textarea)`, which would be the "ancestor selector", instead of e.g. `.elm-label:has(.elm-textarea)`, which would be called "parent selector".